lmfit/parameter.py

```diff
@@ -95,7 +109,8 @@ def update(self, other):
         if not isinstance(other, Parameters):
             raise ValueError(f"'{other}' is not a Parameters object")
         self.add_many(*other.values())
-        for sym in other._asteval.user_defined_symbols():
+        usersyms = clean_np2_symbols(other._asteval.user_defined_symbols())
+        for sym in usersyms:
```

Comment:
This is fine, but I would prefer to leave the original in commented form, with a ```FIXME``` note to revert to it as soon as `asteval` is updated. Unless you will remember which changes to undo as I might not without looking back to the commit history ;)

Reply:
will do.